pkgs/desktops/gnome-3/3.20/devtools/nemiver/bool_slot.patch

```diff
@@ -0,0 +1,13 @@
+--- a/src/dbgengine/nmv-dbg-common.h	2014-07-09 10:36:05.000000000 +0200
```

Comment:
The first patch comes from https://bugs.debian.org/cgi-bin/bugreport.cgi?att=1;bug=822502;filename=nemiver_0.9.6-1ubuntu1.debdiff;msg=10 but I didn't manage to use it directly.
I wrote the second patch from scratch.